assets/router/service-internal.yaml

```diff
@@ -4,13 +4,8 @@ kind: Service
 apiVersion: v1
 metadata:
   # Name and Annotations are set at runtime.
-  namespace: openshift-ingress
-  labels:
-    app: router
 spec:
   type: ClusterIP
-  selector:
-    app: router
```

Comment:
Recent changes are hinting us that we don't have much value in creating assets anymore.
Probably we should ditch assets and create objects internally in the code.

Reply:
If we keep the assets, maybe we could at least use more abstract names, such as `ClusterIPService`, similar to how we have `LoadBalancerService`.  I wouldn't object to removing the assets entirely though.